src/components/example/EForm.tsx

```
@@ -0,0 +1,93 @@
+                            <FormMessage />
+                        </FormItem>
+                    )}
+                />
+                <AlertDialog>
+                    <AlertDialogTrigger asChild>
+                        <Button type="submit">Submit</Button>
+                    </AlertDialogTrigger>
+                    <AlertDialogContent>
+                        <AlertDialogHeader>
+                            <AlertDialogTitle>Are you absolutely sure?</AlertDialogTitle>
+                            <AlertDialogDescription>
```

ESLint

[error] Delete `··············`

(prettier/prettier)